website/src/pages/index.tsx

```diff
@@ -8,7 +8,7 @@
  * @oncall web_perf_infra
```

Comment:
> Integrate TerminalReplay component for improved presentation

It appears the changes here has nothing to do with the `TerminalReplay ` component